src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -712,6 +712,9 @@ export default class ConvasController {
         if (aircraft.category === FLIGHT_CATEGORY.DEPARTURE) {
             return;
         }
```

Comment:
Something like `GameController.shouldUseTrailingSeparatorFor(aircraftModel)` I think is what he's getting at. Some way to combine the two if statements into a single check of a boolean method so it's more concise here.